pubspec.yaml

```
@@ -52,4 +51,4 @@ flutter:
 
   assets:
     - assets/icons/
-    - env/
+    - env/
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)